cli/Sources/TuistGenerator/Mappers/ResourcesProjectMapper.swift

```diff
@@ -376,6 +395,33 @@ public class ResourcesProjectMapper: ProjectMapping { // swiftlint:disable:this
     }
 }
 
+extension BuildableFolder {
+    fileprivate var containsSourceLikeFiles: Bool {
```

Comment:
would probably make this a method instead of a variable since needs to go through all files